src/test/scala/org/clulab/wm/eidos/text/eval6/TestDoc2.scala

```diff
@@ -1,27 +1,190 @@
+      tester.test(markets) should be (successful)
+      tester.test(families) should be (successful)
+
+      // TODO: we really need to decide how to handle nested events!! (we should have "families from accessing food")
+      tester.test(EdgeSpec(prices, Causal, families)) should be (successful)
+      tester.test(EdgeSpec(roads, Causal, families)) should be (successful)
+      tester.test(EdgeSpec(markets, Causal, families)) should be (successful)
     }
   }
+
+  { // Paragraph 5
+    val text = """
+                 |Food insecurity and conflict are also forcing many families to leave South Sudan for neighbouring countries.
+               """
+    val tester = new Tester(text)
+
+    behavior of "TestDoc2 Paragraph 5"
+
+    ignore should "have correct output #5" taggedAs(Somebody) in {
+      val foodInsecurity = NodeSpec("Food insecurity")
+      val conflict = NodeSpec("conflict")
+      val fams = NodeSpec("families", Quant("many"))
+
+      tester.test(foodInsecurity) should be (successful)
+      tester.test(conflict) should be (successful)
+      tester.test(fams) should be (successful)
+
+      // TODO: we really need to decide how to handle nested events!! (we should have "families to leave")
+      tester.test(EdgeSpec(foodInsecurity, Causal, fams)) should be (successful)
+      tester.test(EdgeSpec(conflict, Causal, fams)) should be (successful)
+    }
+  }
+
+  { // Paragraph 6
+    val text = """
+                 |“The levels of malnutrition among children continue to be truly alarming,” said Mahimbo Mdoe,
+                 |UNICEF’s Representative in South Sudan.
+               """
+    val tester = new Tester(text)
+
+    behavior of "TestDoc2 Paragraph 6"
+
+    ignore should "have correct output #6" taggedAs(Somebody) in {
+      val malnutrition = NodeSpec("levels of malnutrition", Quant("truly", "alarming"))
+
+      tester.test(malnutrition) should be (successful)
+    }
+  }
+
+  { // Paragraph 7
+    val text = """
+                 |We have started ramping up food and nutrition support, but much more is needed to keep things
+                 |from deteriorating even further during the lean season,” said WFP Country Director Joyce Luma.
+               """
+    val tester = new Tester(text)
+
+    behavior of "TestDoc2 Paragraph 7"
+
+    ignore should "have correct output #7" taggedAs(Somebody) in {
+      val support = NodeSpec("food and nutrition support", Inc("ramping"))
+
+      tester.test(support) should be (successful)
+    }
+  }
+
+  { // Paragraph 8
+    val text = """
+                 |“We are now seeing sharp spikes of need in new areas, such as Eastern Equatoria or Western
+                 |Bahr el-Ghazal, where malnutrition rates in some places are reaching dangerous levels.
+               """
+    val tester = new Tester(text)
+
+    behavior of "TestDoc2 Paragraph 8"
+
+    ignore should "have correct output #8" taggedAs(Somebody) in {
+      val need = NodeSpec("need", Inc("sharp", "spikes"))
```

Comment:
I think "sharp" and "spikes" should be switched...?

Reply:
and I'd argue there's a Correlation event between them -- btw -- I think I'm going to move Correlation to UndirectedRelations in the taxonomy (FYI)

Reply:
and re: malnutrition -- I think we don't know if Inc or Dec (without inference), so I'd argue a Quant or Unmarked (if we do move to those)